src/routes/index.ts

```diff
@@ -308,4 +313,15 @@ router.route(config.UPDATE_CHECK_YOUR_ANSWERS_URL)
   .get(updateCheckYourAnswers.get)
   .post(updateCheckYourAnswers.post);
 
+router.route(config.UPDATE_BENEFICIAL_OWNER_OTHER_URL)
+  .all(authentication, navigation.hasBeneficialOwnersStatement)
+  .get(updateBeneficialOwnerOther.get)
+  .post(...validator.beneficialOwnerOther, checkValidations, updateBeneficialOwnerOther.post);
+
+router.route(config.UPDATE_BENEFICIAL_OWNER_OTHER_URL + config.ID)
+  .all(authentication, navigation.hasBeneficialOwnersStatement)
+  .get(updateBeneficialOwnerOther.getById)
+  .post(...validator.beneficialOwnerGov, checkValidations, updateBeneficialOwnerOther.update);
```

Comment:
Think checkValidations is wrong for us. Do we need to check for entity_number rather than entity_name?

Reply:
`navigation.hasBeneficialOwnersStatement` cannot be used here as it refers to Registration journey and checks a key which is not implemented in Update journey, for now can you use `navigation.hasUpdatePresenter`, till the page has been implemented in Update journey

Reply:
Looks like similar check has been used in the gov page ?